openedx_external_enrollments/external_enrollments/viper_external_enrollment.py

```diff
@@ -83,6 +84,41 @@ def _get_enrollment_url(self, course_settings):
         """
         return settings.OEE_VIPER_API_URL
 
+    def _refresh_api_keys(self):
+        """
+        Method to execute a post request to viper's API, this will refresh the API key.
```

Comment:
@ivanvgh , I see that `REFRESH_API_KEY_ACTION = 'resetApiKeyExpirationDate'`, so It is convenient to be more consistent saying that the  `_refresh_api_keys` is refreshing the Expiration Date.
```suggestion
        Method to execute a post request to viper's API, this will refresh the API key Expiration Date.
```